authentication_document.py

```diff
@@ -462,7 +462,7 @@ def _update_collection_size(self, library, sizes):
                     CollectionSummary.set(library, None, unknown_size)
                 )
         except ValueError, e:
-            return INVALID_INTEGRATION_DOCUMENT.detailed(e.message)
+            return INVALID_INTEGRATION_DOCUMENT.detailed(unicode(e))
```

Comment:
This should be the same as `str(e)` which we've been doing in `circulation`, right?

Reply:
I have given this some thought and I think either one is going to work. In problem_detail.py, when the problem detail is converted to JSON, almost the first thing we do is convert the detail to Unicode. When we convert to Python 3, str(e.message) and unicode(e.message) will both end up as str(e.message), meaning a Unicode string.